src/Maestro/SubscriptionActorService/PullRequestActor.cs

```diff
@@ -1011,6 +1056,11 @@ private int RemovePRDescriptionSection(string sectionStartMarker, string section
             return description.Length;
         }
 
+        private string GetChangesURI(string repoURI, string from, string to)
+        {
+            return $"{repoURI}/compare/{from}..{to}";
```

Comment:
I'm a fan of the `...` dot comparisons instead of the `..` as it conveys all the commits involved, not just the file changes:

for example 
https://github.com/dotnet/arcade/compare/f6192d1...ea8f37e
vs 
https://github.com/dotnet/arcade/compare/f6192d1..ea8f37e

Reply:
I didn't know the ... dot thing, I'll change that